R/pmxClass.R

```diff
@@ -202,7 +202,7 @@ formula_to_text <- function(form) {
 #' @example inst/examples/pmx-settings.R
 #' @export
 pmx_settings <-
-  function(is.draft = TRUE, use.abbrev = FALSE, color.scales = NULL,
+  function(is.draft = TRUE, use.abbrev = TRUE, color.scales = NULL,
```

Comment:
Going forward, we should make sure the inputs are correct;  I think `checkmate` is an easy way to do this.  Use the `assert_logical` function to make sure it is a logical with length 1 without a missing value (which is actually by default a logical)

Reply:
Good point. I've added arg tests for the logical arguments of `pmx_settings` with checkmate as an import. Also a unit test for `pmx_settings` has been added to test the class of its return value.